### PR TITLE
Drop unmaintained daterangepicker package

### PR DESCRIPTION
**Pull request by hluk**
Apart from being unmaintained, there were multiple suspicious package changes (integrity checksum does not match).

JIRA: RHELWF-11724

### CODE REVIEW

**Comment by hluk** [comment]
@mvalik Please review. The package changed again so I'm dropping it. If needed we can replace the UI later with something else, thought I'm not sure how useful the date search is.

File: resultsdb_frontend/static/js/search.js [hunk 1]
```diff
@@ -70,38 +70,23 @@ $('document').ready(function() {
...
-        }
-    }, function(start, end, label) {
-        if (label === 'Custom Range'){
-            //$('#searchrange').val(start.format('MMMM D, YYYY') + ' - ' + end.format('MMMM D, YYYY'));
-            since = start.format('YYYY-MM-DD') + 'T00:00:00,' + end.format('YYYY-MM-DD') + 'T23:59:59';
-        }
-        else{
-            $('#searchrange').val(label);
-            since = start.format('YYYY-MM-DD') + 'T00:00:00,' + end.format('YYYY-MM-DD') + 'T23:59:59';
+    var date_range = function(since) {
```
Comment (mvalik) [inline resultsdb_frontend/static/js/search.js]: Using `var` is obsolete in JavaScript. You should use `let` or `const` instead. Also, variable names should follow `lowerCamelCase` style, such as `dateRange`, which is recommended for naming.

**Review by mvalik — CHANGES_REQUESTED** [review]
Make sure to follow the coding guidelines for JavaScript.

File: resultsdb_frontend/static/js/search.js (hunk 1 above)
Comment (hluk) [reply resultsdb_frontend/static/js/search.js]: I would prefer to keep the code consistent. Do you have a link to a document which says `var` is obsolete?

**Comment by hluk** [comment]
> Make sure to follow the coding guidelines for JavaScript.

Please be more specific.

File: resultsdb_frontend/static/js/search.js (hunk 1 above)
Comment (hluk) [reply resultsdb_frontend/static/js/search.js]: I've changed this to simpler `function date_range(since)`.

Comment (mvalik) [reply resultsdb_frontend/static/js/search.js]: See https://google.github.io/styleguide/jsguide.html#naming about naming and https://google.github.io/styleguide/jsguide.html#features-local-variable-declarations about `var` vs `let` and `const` (if you want to know more why it's discouraged to used `var`: https://plainenglish.io/blog/4-reasons-why-var-is-considered-obsolete-in-modern-javascript (TL;DR `let` is safe in the sense of scoping, (re)declaration, checking whether it was defined or not, while `var` is not safe))

Comment (hluk) [reply resultsdb_frontend/static/js/search.js]: Feel free to create an issue to refactor the js file and perhaps add some checks. Though I currently see it as a minor priority.

**Pull request merged**